util/util.go

```diff
@@ -117,10 +118,32 @@ func GetProviders() []provider.Provider {
 			if key == "telegram" && c == "chatid" && len(strings.TrimSpace(v.(string))) > 0 {
 				telegram[1] = true
 			}
+			if key == "email" && c == "from" && len(strings.TrimSpace(v.(string))) > 0 {
```

Comment:
I think there's a better way (more readable) for this validation

Reply:
@abahmed will be refactored in another PR